lib/lorekeeper/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Lorekeeper
-  VERSION = '2.3.0'
+  VERSION = '2.2.1'
```

Comment:
You don't need to downgrade this version. We have already released:

```suggestion
  VERSION = '2.3.0'
```